### DISCUSSION TITLE
LIGHTDM_CONF="greeter-hide-users=true" not working

### DISCUSSION THREAD

**Discussion by Madtoyit** [ltsp/ltsp#278] [Q&A]
I'm on ubuntu 20.04 and ltsp 20.04.
We have problem with LIGHTDM_CONF="greeter-hide-users=true" seems not to work.
In guide ltsp and how we have see in other posts on community, for disable the list of user on login screen on client is only necessary in etc/ltsp/ltsp.conf the parameter LIGHTDM_CONF="greeter-hide-users=true".
after any change we do only "ltsp initrd"
We tried to put the parameter in all section [server common clients] olso in a specific [macaddress] section but no change for the login screen. 
We have olso used "POST_INIT_SHELL=bash" for check the correct update of the ltsp.conf and is it.

I post our code ltsp.conf, so what we are missing?
tks

```shell
# /bin/sh -n
# LTSP configuration file
# Documentation=man:ltsp.conf(5)

# The special [server] section is evaluated only by the ltsp server
[server]
# LIGHTDM_CONF="greeter-hide-users=true"
# Enable NAT on dual NIC servers
# NAT=1
# Provide a full menu name for x86_32.img when `ltsp ipxe` runs
# IPXE_X86_32_IMG="Debian Buster"

# The special [common] section is evaluated by both the server and ltsp clients
[common]
# LIGHTDM_CONF="greeter-hide-users=true"
# POST_INIT_SHELL=bash
# Specify an alternative TFTP_DIR
# TFTP_DIR=/var/lib/tftpboot

# In the special [clients] section, parameters for all clients can be defined.
# Most ltsp.conf parameters should be placed here.
[clients]
# LIGHTDM_CONF="greeter-hide-users=true"
# POST_INIT_SHELL=bash
# POST_INIT_FIX_PAMLTSP="sed 's/gnome-keyring-daemon/unused/g' -i /usr/share/ltsp/client/login/pamltsp"
# Specify an /etc/fstab line for NFS home; note this is insecure
# FSTAB_HOME="server:/home /home nfs defaults,nolock 0 0"

# MAC address, IP address, or hostname sections can be used to apply settings
# to specific clients.

[e8:6a:64:ba:03:87]
LIGHTDM_CONF="greeter-hide-users=true"
#POST_INIT_SHELL=bash

[d6:0c:03:8e:be:18]
LIGHTDM_CONF="greeter-hide-users=true"
# POST_INIT_SHELL=bash
HOSTNAME=bubuca
# Include parameters from another section, defined below
# INCLUDE=crt_monitor

# Shell "case" expressions can be used in MAC, IP, or hostname sections.
# This matches all Raspberry Pi MAC addresses.
[b8:27:eb:*|dc:a6:32:*]
# FSTAB_BOOT="/dev/mmcblk0p1  /boot  vfat  defaults  0  2"

# If you have proper DNS, you can use hostname sections
[administrator-pc]
# Only allow the administrator to log in to this client
# PWMERGE_SUR="administrator"

# You can also group parameters into named sections and INCLUDE= them elsewhere
[crt_monitor]
# Force EDID and resolution to 1024x768 for clients with old CRT monitors
X_HORIZSYNC="28.0-87.0"
X_VERTREFRESH="43.0-87.0"
X_MODES='"1024x768" "800x600" "640x480"'
```

**Comment by alkisg** [comment]
What's the output of this command, on the client?

```shell
grep -r . /etc/lightdm/
```

To put code in github, use this syntax:
 ````shell
 ```shell
code goes here
```
````

**Comment by Madtoyit** [comment]
Hi,
there is no lightdm directory
this is the output:
```shell
user03@nbmiolenovo129:~$ grep -r . /etc/lightdm/
grep: /etc/lightdm/: File o directory non esistente
user03@nbmiolenovo129:~$ 
```

**Comment by alkisg** [comment]
You can't configure lightdm if you don't have it installed! :)

Maybe you're using Ubuntu GNOME, which uses gdm? What's the output of this command?

```shell
echo /etc/*dm* /usr/share/xsessions/*
```

**Comment by Madtoyit** [comment]
and so:

```shell
user03@nbmiolenovo129:~$ echo /etc/*dm* /usr/share/xsessions/*
/etc/gdm3 /etc/idmapd.conf /usr/share/xsessions/ubuntu.desktop
user03@nbmiolenovo129:~$ 
```

**Comment by alkisg** [comment]
Indeed, you have Ubuntu GNOME, so use GDM3_CONF, not LIGHTDM_CONF.

**Comment by Madtoyit** [comment]
Tks but it doesn't
GDM3_CONF="greeter-hide-users=true"
i have always the user list.
Also in the man of ltsp.conf there isn't that parameter.
Can you please verify that. tks

**Comment by alkisg** [comment]
Sure it is, just search for GDM3_CONF: https://ltsp.org/man/ltsp.conf/

GDM doesn't provide a "greeter-hide-users=true" option. The available LightDM and GDM configuration options are documented in their man pages, not in LTSP.

Read this discussion on how to disable the user list on GDM:
https://github.com/ltsp/community/issues/37#issuecomment-547285647

**Reply by Soylor** [reply]
The link does not work. Any alternative link, please?
Thank yoi

**Reply by Soylor** [reply]
I got an alternative to #37 (page is not working)
[https://help.gnome.org/admin/system-admin-guide/stable/login-userlist-disable.html.en]

**Comment by Madtoyit** [comment]
Tks alkisg,
we followed # 37 and now the list of users is hidden. perfect. thanks
